app/lib/actions/katello/content_view/node_metadata_generate.rb

```diff
@@ -7,6 +7,7 @@ def resource_locks
         end
 
         def humanized_name
+          return _('Unknown') if input.nil?
           _("Generate and Synchronize Capsule Metadata for %s") % input[:environment_name]
         end
```

Comment:
For all of these, something like:
```
def humanized_name
  if input.try([], :environment_name)
    _("Generate and Synchronize Capsule Metadata for %s") % input[:environment_name]
  else
   _("Generate and Synchronize Capsule Metadata")
  end
end
```

would be much preferred, as then the user would still see what the type of task is.